test/integration/suites/evict-agent/08-delete-agent

```diff
@@ -1,21 +0,0 @@
-#!/bin/bash
-
-log-debug "deleting agent..."
```

Comment:
I've added some checks that the agents was able to re-attest. Note that I've had to make a bunch of changes in this test because it was unreliable on my machine. E.g. a server was brought up and then the test wouldn't wait for it to be available before trying to use it.